src/main/webapp/WEB-INF/jsp/submitFeedback.jsp

```diff
@@ -39,37 +40,62 @@
 		</div>
 	    </c:if>
 	</spring:bind>
-	<br>
-	<p id="${n}answer">
-      	<form:radiobutton id="yes" path="like" value="YES"/>
-      	<label class="portlet-form-field-label"><spring:message code="feedback.answer.yes"/></label>
-      	<form:radiobutton id="no" path="like" value="NO"/>
-      	<label class="portlet-form-field-label"><spring:message code="feedback.answer.no"/></label>
-      	<form:radiobutton id="maybe" path="like" value="MAYBE"/>
-      	<label class="portlet-form-field-label"><spring:message code="feedback.answer.maybe"/></label>
-    </p>
+
+    <div data-role="fieldcontain">
+    <fieldset data-role="controlgroup" id="${n}answer">
+          <input type="radio" id="yes" name="like" value="YES"/>
+          <label for="yes" class="portlet-form-field-label"><spring:message code="feedback.answer.yes"/></label>
+          <input type="radio" id="no" name="like" value="NO"/>
+          <label for="no" class="portlet-form-field-label"><spring:message code="feedback.answer.no"/></label>
+          <input type="radio" id="maybe" name="like" value="MAYBE"/>
+          <label for="maybe" class="portlet-form-field-label"><spring:message code="feedback.answer.maybe"/></label>
+    </fieldset>
+    </div>
 
  	<p> 		
  		<label class="portlet-form-field-label"><spring:message code="feedback.form.suggestion"/></label>
- 		<textarea id="${n}feedback" path="feedback" name="feedback" rows="${feedbackRows}" style="width:${feedbackWidth}"></textarea> 		
+         <textarea id="${n}feedback" name="feedback" rows="${feedbackRows}" style="width:${feedbackWidth}"></textarea>
  	</p>
  	<div id="${n}limit" style="margin-bottom: 12px;"></div>
 
- 	<p>
-        <form:checkbox path="anonymous" value="true"/>
-		<label class="portlet-form-field-label"><spring:message code="feedback.form.anonymous"/></label>
-	</p>
+     <p>
+        <fieldset data-role="controlgroup">
+            <input type="checkbox" id="${n}anonymous" name="anonymous" value="true"/>
+            <label for="${n}anonymous" class="portlet-form-field-label"><spring:message code="feedback.form.anonymous"/></label>
+        </fieldset>
+    </p>
 
  	<input id="${n}useragentstring" type="hidden" name="useragent"/>
  	<input id="${n}feedbacktabname" type="hidden" name="tabname"/>
 
     <p>
-       <button type="submit" id="submit" class="feedback-submit-button" disabled="disabled"><spring:message code="feedback.form.submit"/></button>
+       <input type="submit" id="${n}submitfeedback" class="feedback-submit-button ui-btn-hidden" disabled="disabled" aria-disabled="true" value="<spring:message code="feedback.form.submit"/>">
     </p>
 
 </form:form>
-
-<script type="text/javascript">
+</div>
+
+<script type="text/javascript"><rs:compressJs>
+   // Enable submitFeedback work here only on Mobile
+    var $ = up.jQuery;
```

Comment:
for consistency with uPortal JavaScript and to avoid global naming conflicts, this should be wrapped in
``` javascript
up.jQuery(function() {
    // Enable submitFeedback work here only on Mobile
    var $ = up.jQuery;
    // rest of code goes here
});
```